server/server.go

```diff
@@ -393,7 +393,7 @@ func New(
 				}
 				logrus.Tracef("default apparmor profile contents: %s", profileContent)
 			}
-		} else {
+		} else if config.ApparmorProfile != "unconfined" {
```

Comment:
I would prefer in having documentation around this in those places:
https://github.com/cri-o/cri-o/blob/544c64b0afbbf3e8944709313a604cfea5a6c508/pkg/config/template.go#L141-L143
https://github.com/cri-o/cri-o/blob/544c64b0afbbf3e8944709313a604cfea5a6c508/internal/pkg/criocli/criocli.go#L457-L459

Afterwards the CLI config can be generated via:
```
> make clean all
> sudo make docs-generation completions
```

Reply:
Can you elaborate on what you'd like? To me this seems like an implementation detail of crio.

Reply:
Yep my intention was to tell the end user how CRI-O behaves if we set `apparmor_profile = "unconfined"`. So it means that AppArmor will be disabled for workloads not specifying an own profile via the annotation, right?

Reply:
I'm not sure what "an own profile via the annotation" means; if a task's apparmor label is set to "unconfined" in the kernel, apparmor will not deny anything.

Reply:
Yes, but this applies only if we do not specify any profile via Kubernetes Pod’s metadata annotation:
https://kubernetes.io/docs/tutorials/clusters/apparmor/#securing-a-pod

This means if the user specifies a custom AppArmor profile via a k8s pod, then the `apparmor_profile = "unconfined"` option will have no effect:
https://github.com/cri-o/cri-o/blob/544c64b0afbbf3e8944709313a604cfea5a6c508/server/container_create.go#L684-L689

This should be documented somehow if this is the intended behavior :)

Reply:
I don't really know if it is the intended behavior or not, but I pushed some docs to that effect :)

Reply:
Hehe, thank you :pray: 

I think it could be the intended behavior, at least that is something _I_ would expect to happen. :grin: